packages/flutter/lib/src/gestures/binding.dart

```diff
@@ -226,6 +339,38 @@ mixin GestureBinding on BindingBase implements HitTestable, HitTestDispatcher, H
       pointerSignalResolver.resolve(event);
     }
   }
+
+  void _handleSampleTimeChanged() {
+    if (!locked) {
+      _flushPointerEventQueue();
+    }
+  }
+
+  // Resampler used to filter incoming pointer events when resampling
+  // is enabled.
+  late final _Resampler _resampler = _Resampler(
```

Comment:
It's immediately initialized. It seems that `late` is not necessary?

Reply:
I needed this to be able to pass "_handlePointerEvent, _handleSampleTimeChanged" to this ctor. Let me know if there's a better way to solve that.

Reply:
Sorry I misunderstood the keyword.